client.go

```diff
@@ -400,13 +401,32 @@ func (c *Client) Mail(from string, opts *MailOptions) error {
 			return errors.New("smtp: server does not support SMTPUTF8")
 		}
 	}
+	if _, ok := c.ext["DSN"]; ok && opts != nil {
+		switch opts.Return {
+		case ReturnFull, ReturnHeaders:
+			cmdStr += "%s"
```

Comment:
I'm not a fan of using params and %-formatting here. Can we use a `strings.Builder` to build the command string, and not use the %-formatting params at all? (IOW: `c.cmd(25, "%s", sb.String())`).

Reply:
Thanks for suggestion.  Done!